features/lwipstack/LWIPStack.cpp

```diff
@@ -195,6 +195,39 @@ nsapi_error_t LWIP::get_dns_server(int index, SocketAddress *address, const char
     return NSAPI_ERROR_NO_ADDRESS;
 }
 
+nsapi_error_t LWIP::add_dns_server(const SocketAddress &address, const char *interface_name)
+{
+    int index;
+    nsapi_addr_t addr = address.get_addr();
+    const ip_addr_t *ip_addr_move;
+    ip_addr_t *ip_addr;
+
+    convert_mbed_addr_to_lwip(ip_addr, &addr);
```

Comment:
and this (and later in function) should then use &ip_addr.

Reply:
fixed